include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -424,8 +428,9 @@ __radix_sort_scan_submit(_ExecutionPolicy&& __exec, ::std::size_t __scan_wg_size
                 // find borders of a region with a specific bucket id
                 sycl::global_ptr<_CountT> __begin = __count_rng.begin() + __scan_size * __self_item.get_group(0);
                 // TODO: consider another approach with use of local memory
-                sycl::ONEAPI::exclusive_scan(__self_item.get_group(), __begin, __begin + __scan_size, __begin,
-                                             _CountT(0), sycl::ONEAPI::plus<_CountT>{});
+                oneapi::dpl::__par_backend_hetero::__joint_exclusive_scan(__self_item.get_group(), __begin,
+                                                                          __begin + __scan_size, __begin, _CountT(0),
+                                                                          sycl::ONEAPI::plus<_CountT>{});
```

Comment:
Lets add a template type alias for "plus", like this
```
template <typename  _Tp>
using __sycl_plus = sycl::ONEAPI::plus<_Tp>;
```

Reply:
Your comment is useful, especially in scope of [this patch](https://github.com/intel/llvm/commit/d703f578fcf7e8fb4716846bdd115add128a345b) by DPC++ compiler, because they change ONEAPI namespace there. But I suggest applying your comment later in a separate review, because we do not have a macro to detect the change by the compiler for now.

Reply:
up to you